Forwards.hpp

```diff
@@ -135,6 +135,8 @@ enum Color {
   COLOR_INVALID = 3u
 };
 
+enum SymbolType{FUNC, PRED, TYPE_CON};
```

Comment:
Could this be `enum class` without too much trouble?

Reply:
It can be, and making the change caught a bug right out of the box!